sumologic/resource_sumologic_monitors_library_monitor.go

```diff
@@ -651,6 +660,34 @@ var logsAnomalyTriggerConditionSchema = map[string]*schema.Schema{
 	}),
 }
 
+var metricsAnomalyTriggerConditionSchema = map[string]*schema.Schema{
```

Comment:
Later, we should refactor and reuse `metricsAnomalyTriggerConditionSchema` within `logsAnomalyTriggerConditionSchema`.